README.md

```diff
@@ -169,6 +169,18 @@ Usage: mix coveralls.post <Options>
     -s (--sha)          Commit SHA (required when not using Travis)
     --build             Service number ('BUILDS' column at coveralls.io page)
     --parallel          coveralls.io 'parallel' option (See coveralls.io API Reference)
+    --flagname         coveralls.io 'flag_name' option (See coveralls.io API Reference)
```

Comment:
conflict marker here - also this bit of the readme is specifically talking about the `post` task, as this is common to all tasks we should have it in the section above (line 131)

Reply:
https://github.com/duffelhq/excoveralls/pull/1/commits/9282870949692b965b8c2ae3e9102755256f43bf

Fixed here :D